_toc.rst

```diff
@@ -1,7 +1,8 @@
 .. toctree::
    :includehidden:
    :maxdepth: 99
-
+   
+   index
```

Comment:
so this will put the top-leveel heading as the book title, and then the introduction nested underneath it.  

If you use  Introduction <index>   ... then the top-level of the contents will say Introduction.  Then, in the index.rst file, you can get rid of the introduction header, and nest the little paragraph directly under the book name.